src/Routes.js

```diff
@@ -11,12 +12,11 @@ const paths = {
   workflows: '/workflows'
 };
 export const Routes = () => {
-  const approvalAdmin = useSelector(
-    ({ rolesReducer: { approvalAdmin }}) => approvalAdmin);
+  const { roles: userRoles } = useContext(UserContext);
 
   return <Suspense fallback={ <RequestLoader /> }>
     <Switch>
-      { approvalAdmin && <Route path={ paths.workflows } component={ Workflows }/> }
+      { isApprovalAdmin(userRoles) && <Route path={ paths.workflows } component={ Workflows }/> }
```

Comment:
I think we should keep the route. Just hide the rendered component behind some permission check and let the user know they don't have the necessary permissions.